### PR TITLE
Player eye height: Make this a settable player object property

### PR DESCRIPTION
**Pull request by paramat**
screenshot_20171103_195124

^ I'm 1 pixel tall

Implements #6216 
MTGame PR to be merged with this https://github.com/minetest/minetest_game/pull/1939

### CODE REVIEW

File: src/script/common/c_content.cpp [hunk 1]
```diff
@@ -266,6 +266,8 @@ void read_object_properties(lua_State *L, int index,
 	if (getfloatfield(L, -1, "stepheight", prop->stepheight))
 		prop->stepheight *= BS;
 	getboolfield(L, -1, "can_zoom", prop->can_zoom);
+	if (getfloatfield(L, -1, "eye_height", prop->eye_height))
+		prop->eye_height *= BS;
```
Comment (paramat) [inline src/script/common/c_content.cpp]: maybe do not * BS here?

**Comment by paramat** [comment]
Ok updated.

**Comment by paramat** [comment]
Another bugfix for 'get properties'.

**Comment by lhofhansl** [comment]
Note that this would have sent to the server as well in order to do correct server side occlusion culling.

**Comment by ExeterDad** [comment]
This will be awesome if merged! My imagination just went through the roof with clever uses for this. 👏

**Comment by paramat** [comment]
lhofhansl thanks will do.

**Comment by lhofhansl** [comment]
@ExeterDad Agreed. I have been wanting to have a shrinking potion or an ant-world and similar :)
(I guess both require an eye-height change and a FOV change)

**Comment by paramat** [comment]
@lhofhansl see updated commit, not sure if this is enough?

**Comment by paramat** [comment]
Also wondering if tthis has been hardcoded for clientside occlusion culling, can't track down the code yet.

**Comment by paramat** [comment]
I've spent an hour tracking through code, i think this is everything now.

**Comment by paramat** [comment]
4c40e0775ca564296a56f72ff3adce50ba925b0c

**Pull request closed without merging**